database/user.go

```diff
@@ -36,12 +36,13 @@ var (
 
 // User is the database representation of a user.
 type User struct {
-	ID     sql.NullInt64  `sql:"id"`
-	Name   sql.NullString `sql:"name"`
-	Token  sql.NullString `sql:"token"`
-	Hash   sql.NullString `sql:"hash"`
-	Active sql.NullBool   `sql:"active"`
-	Admin  sql.NullBool   `sql:"admin"`
+	ID        sql.NullInt64  `sql:"id"`
+	Name      sql.NullString `sql:"name"`
+	Token     sql.NullString `sql:"token"`
+	Hash      sql.NullString `sql:"hash"`
+	Favorites sql.NullString `sql:"favorites"`
```

Comment:
🤔 I don't think we want this to be a single string. I think we want it to be `pq.StringArray`.

This follows the pattern we have for the secrets `database.Secret.Images` and `database.Secret.Events` fields.

https://github.com/go-vela/types/blob/master/database/secret.go#L53-L54